### PR TITLE
delete post

### PR DESCRIPTION
**Pull request by ghost**
## Description
Delete Post
## Reference Requirements
- Task ID: #57 

## Reference Isuees
#{issue_id}

 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

```
mysql> explain select count(*) as aggregate from `posts` \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: posts
   partitions: NULL
         type: index
possible_keys: NULL
          key: posts_user_id_foreign
      key_len: 4
          ref: NULL
         rows: 13
     filtered: 100.00
        Extra: Using index
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified
```
```
mysql> explain select * from `posts` limit 10 offset 0 \G;
*************************** 1. row ***************************
           id: 1
  select_type: SIMPLE
        table: posts
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 13
     filtered: 100.00
        Extra: NULL
1 row in set, 1 warning (0.00 sec)

ERROR: 
No query specified

```

### CODE REVIEW

File: app/Http/Controllers/Admin/PostsController.php [hunk 1]
```diff
@@ -18,4 +18,21 @@ public function index()
         $posts = Post::paginate(Post::PAGINATE);
         return view('admin.post.index', ['posts'=>$posts]);
     }
+    /**
```
Comment (at-sonle) [inline app/Http/Controllers/Admin/PostsController.php]: thêm một inline

File: resources/views/admin/post/index.blade.php [hunk 2]
```diff
@@ -49,7 +49,9 @@
                     <td>{{ $post->type }}</td>
                     <td>{{ $post->status }}</td>
                     <td>
-                      <form>
+                      <form method="post" action="{{route('admin.post.delete',$post->id)}}">
```
Comment (at-sonle) [inline resources/views/admin/post/index.blade.php]: thêm một space sau dấu ,

**Comment by at-sonle** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🕋

**Pull request merged**